models/User.php

```diff
@@ -593,11 +593,9 @@ private function getMergedWebsiteTranslationLanguagesEdit(): array
     public function getAllowedLanguagesForEditingWebsiteTranslations(): ?array
     {
         $mergedWebsiteTranslationLanguagesEdit = $this->getMergedWebsiteTranslationLanguagesEdit();
-        if (empty($mergedWebsiteTranslationLanguagesEdit) || $this->isAdmin()) {
-            $mergedWebsiteTranslationLanguagesView = $this->getMergedWebsiteTranslationLanguagesView();
-            if (empty($mergedWebsiteTranslationLanguagesView)) {
-                return Tool::getValidLanguages();
-            }
+         if ((empty($mergedWebsiteTranslationLanguagesEdit) && empty($this->getMergedWebsiteTranslationLanguagesView())) 
+            || $this->isAdmin()) { 
```

Comment:
empty() isn't necessary because the variables are always defined
```suggestion
         if (
             (!$mergedWebsiteTranslationLanguagesEdit && !$this->getMergedWebsiteTranslationLanguagesView()) ||
             $this->isAdmin()
         ) { 
```